metadata-service/restli-servlet-impl/src/main/java/com/linkedin/metadata/resources/entity/AspectResource.java

```diff
@@ -97,7 +97,7 @@ public Task<AnyRecord> get(@Nonnull String urnStr, @QueryParam("aspect") @Option
     return RestliUtil.toTask(() -> {
       final VersionedAspect aspect = _entityService.getVersionedAspect(urn, aspectName, version);
       if (aspect == null) {
-        throw RestliUtil.resourceNotFoundException();
+        throw RestliUtil.resourceNotFoundException(String.format("Did not find urn: %s aspect: %s version: %s", urn, aspectName, version));
```

Comment:
concern: adding the urn to exceptionmessages, can leak PII

Reply:
The URN in question is passed in as input parameter. So the caller already has knowledge of the URN and thus the PII. The URN is being logged few lines above either way. It just makes it possible to connect the URN with the exception.